subsys/net/lib/sockets/sockets.c

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if defined(CONFIG_NET_DEBUG_SOCKETS)
+#define SYS_LOG_DOMAIN "net/sock"
+#define NET_LOG_ENABLED 1
+#endif
+
+#include <net/net_context.h>
+#include <net/net_pkt.h>
+#include <net/socket.h>
```

Comment:
This would also be
`#include <net/zsocket.h>`
as it's calling native Zephyr socket APIs, rather than POSIX socket, close, recv, etc.